frontend/coprs_frontend/coprs/forms.py

```diff
@@ -676,6 +676,12 @@ class CoprForm(BaseForm):
         default=None,
     )
 
+    storage = wtforms.SelectField(
+        "Admin only - what storage should be set for new projects",
+        choices=[(x, x) for x in ["backend", "pulp"]],
+        validators=[wtforms.validators.Optional()],
+    )
+
```

Comment:
Can we hide this option for normal users so they can't see it?

Reply:
I may have missed something but users shouldn't see it already. When creating/editing a project in the web UI, the option shouldn't be there for either admins or normal users. For API, it will probably have to be visible?

Also, I just realized that I forgot to add the field into the API schema and it works regardless. Is this expected @nikromen? I will add it though :-)

Reply:
>  I will add it though :-)

Done